lms/envs/aws.py

```diff
@@ -759,6 +759,7 @@
 ##### GOOGLE ANALYTICS IDS #####
 GOOGLE_ANALYTICS_ACCOUNT = AUTH_TOKENS.get('GOOGLE_ANALYTICS_ACCOUNT')
 GOOGLE_ANALYTICS_LINKEDIN = AUTH_TOKENS.get('GOOGLE_ANALYTICS_LINKEDIN')
+GOOGLE_ANALYTICS_MEASUREMENT_ID = AUTH_TOKENS.get('GOOGLE_ANALYTICS_MEASUREMENT_ID')
```

Comment:
Maybe it will be better to use the default value, like:
`AUTH_TOKENS.get('GOOGLE_ANALYTICS_MEASUREMENT_ID', GOOGLE_ANALYTICS_MEASUREMENT_ID)`

aws.py imports `GOOGLE_ANALYTICS_MEASUREMENT_ID` variable from common.py (which is 'None'), am I right?